### DISCUSSION TITLE
Welcome to fio Discussions!

### DISCUSSION THREAD

**Discussion by axboe** [axboe/fio#1634] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by HankB** [comment]
HankB here - dev of embedded systems, fascinated with Raspberry Pis and rabid Penguinista.

I have a problem. I'm obsessed with storage speed. There, I said it! Processor and display speed also matter but long ago I realized that storage speed can make a significant difference in the performance of a system. `fio` has been one of my go-to benchmarks to evaluate that. When this popped up in the Github landing page, I had to follow and ... here I am.

One of the things I want to do is evaluate the options I'm using to determine if they make sense for my usage. I need to poke around for that but for now I'll just say "Hi."

Thanks!
